lib/aruba/api.rb

```diff
@@ -98,12 +98,54 @@ def directory?(file)
       File.directory? expand_path(file)
     end
 
+    # Check if path is absolute
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def absolute?(path)
+      Pathname.new(path).absolute?
+    end
+
+    # Check if path is relative
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def relative?(path)
+      Pathname.new(path).relative?
+    end
+
     # Return all existing paths (directories, files) in current dir
     #
     # @return [Array]
     #   List of files and directories
     def all_paths
```

Comment:
Oh, I didn't noticed that you mean `all_paths`. This one was added by me to reduce code duplication in `check_file_presence` and in `Then /^(?:a|the) file matching %r<(.*?)> should (not )?exist$/ do |pattern, expect_match|`